features/proposed/FxA-70-kpi-dashboards/README.md

```diff
@@ -0,0 +1,414 @@
+reported on a daily basis,
+separated by new-account and existing-account connections.
+It will look something like this:
+
+---
+![](connection-time-graph.png)
+---
+
+A linked detail dashboard can present
+this data broken down by geographic region,
+as well as timing data for individual steps
+of the setup process.
+This will be designed and implemented
+as follow-up work.
+
+To account for
+failed or aborted connection attempts,
+we will also show the completion rate
+as a percentage
+in the linked detail view.
+
+
+## Details
+
+### Definitions
+
+To avoid ambiguity
+and ensure alignment with metrics
+generated by other teams at Mozilla,
+we'll use the following concrete definitions
+of the data of interest:
+
+* "Active user" refers to a specific FxA user id,
+  and means that we recorded at least one activity event
+  associated with that userid
+  in the time period in question.
+
+* "Active device" refers to a specific FxA device id,
+  and means that we recorded at least one activity event
+  associated with that device id
+  in the time period in question.
+
+* "Active daily users" or "ADU" means
+  the number of distinct user ids
+  for which we have at least one activity event in our logs
+  during a given UTC day.
+
+* "Active monthly users" or "AMU" on a given day means
+  the number of distinct user ids
+  for which we have at least one activity event in our logs
+  during the previous 30 UTC days.
+
+* "Engagement ratio" on a given day means
+  the mean ADU over the previous 7 days,
+  divided by the AMU for that day,
+  expressed as a percentage.
+
+* A userid is a "multi-device user" on a given day
+  if their account had an active device on that UTC day,
+  and had an active device with a different device id
+  in the previous five UTC days.
+
+* "Multi-Device ADU" means
+  the number of distinct user ids
+  which counted as multi-device users on a given UTC day.
+
+* "Multi-Device AMU" on a given day means
+  the number of distinct user ids
+  which counted as multi-device users
+  at any time during the previous 30 UTC days.
+
+* A device is "ready to sync"
+  when it has registered a device metadata record
+  with the FxA auth backend,
+  and it has successfully fetched encryption keys
+  from the server.
+
+* "Time to device connection" means the time elapsed,
+  in seconds,
+  between the device first requesting connection UI
+  from the FxA front-end servers,
+  and the device being ready to sync.
+
+The definition of a "multi-device user" here
+is somewhat arbitrary,
+but is designed to capture
+users who are actively engaged in a "multi-device experience".
+Only counting users with
+multiple devices active on a single day
+would seem too restrictive to
+encompass more casual users,
+while counting devices used weeks apart
+would not seem to capture what's really meant
+by "multi-device experience".
+
+### Data Capture and Storage
+
+We will use AWS Redshift as the canonical store
```

Comment:
@rfk we should chat about this in the 1:1, to get some practice with Redshift we can start sending the verified time in https://github.com/mozilla/fxa-auth-server/blob/9d864ce1ba45f8cd35ca26cba650d07c6932ab6e/lib/routes/account.js#L925 to it. This way we can get a dashboard and queries going, then just switch it to show "time to device connected"